### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by garrensmith**

### CODE REVIEW

File: src/help.js [hunk 1]
```diff
@@ -24,13 +24,16 @@ export default help;
...
+    const helpDir = nmo.config.get('help-directory');
 
     if (!cmd || !nmo.commands[cmd]) {
       help();
       return resolve();
     }
 
-    const path = `${__dirname}/../man/man1/nmo-${cmd}.1`;
+    const path = `${helpDir}/man/man1/${name}-${cmd}.1`;
+    console.log('ppp', path, name, process.cwd());
```
Comment (robertkowalski) [inline src/help.js]: debug output

File: src/nmo.js [hunk 2]
```diff
@@ -49,6 +49,8 @@ nmo.load = function load (opts) {
       .load(opts)
       .then((config) => {
         nmo.config = config;
+        nmo.config.set('name', 'nmo');
+        nmo.config.set('help-directory', `${__dirname}/..`);
```
Comment (robertkowalski) [inline src/nmo.js]: hmmm...

what do you think of adding them as defaults to the nmo conf file that is placed into the home dir?

this smells dangerously for race conditions

Comment (garrensmith) [inline src/nmo.js]: Thats a good idea. I will do that.

Comment (robertkowalski) [inline src/nmo.js]: we might want include a separate template which is then loaded here https://github.com/garrensmith/couchdb-nmo/blob/small-fixes/bin/nmo-cli.js#L24

Comment (robertkowalski) [inline src/nmo.js]: ehm i mean default config file, not template

Comment (robertkowalski) [inline src/nmo.js]: not sure if we need to migrate old other config. the `ini` module should make migration a 4-5 liner

**Comment by robertkowalski** [comment]
i thought about that.

i would suggest to directly use the `name` field from the `package.json` to make nmo reusable.

**Comment by robertkowalski** [comment]
+1

**Pull request merged**